### PR TITLE
Modified readme.md

### PR DESCRIPTION
**Pull request by jbloom22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 [![Join the chat at https://gitter.im/broadinstitute/hail](https://badges.gitter.im/broadinstitute/hail.svg)](https://gitter.im/broadinstitute/hail?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Hail is a framework for scalable genetic data analysis.  Hail is
+Hail is an open-source framework for scalable genetic data analysis.  Hail is
 pre-alpha software and under active development.  Hail is written in
 Scala (mostly) and uses Apache [Spark](http://spark.apache.org/) and
 other Apache Hadoop projects.  If you are interested in getting
-involved in Hail development, email hail@broadinstitute.org.
+involved in Hail development, email hail-dev@broadinstitute.org.
```
Comment (cseed) [inline README.md]: hail-dev is going away.  Should link to discuss/development.

**Comment by jbloom22** [comment]
Here's a larger rewrite of the homepage, ready for feedback.

The gitter links reflect hail and hail-dev as we want them to be, so before merging we should rename hail to hail-dev and create hail.

I think the development forum is a good place for more detailed instructions on collaboration (forking, etc) and best practices.

File: README.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 
 [![Join the chat at https://gitter.im/broadinstitute/hail](https://badges.gitter.im/broadinstitute/hail.svg)](https://gitter.im/broadinstitute/hail?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (cseed) [inline README.md]: Link to hail-is/hail room.  I'm going to delete broadinstitute/hail.

Comment (cseed) [inline README.md]: Same below.

File: README.md [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
-Scala (mostly) and uses Apache [Spark](http://spark.apache.org/) and
-other Apache Hadoop projects.  If you are interested in getting
-involved in Hail development, email hail@broadinstitute.org.
+Hail is an open-source framework for scalable genetic data analysis. To get started using Hail, see the [documentation](https://hail.is/docs/), join the [discussion forum](http://discuss.hail.is), and chat with the Hail team and other users on the [Hail gitter](https://gitter.im/broadinstitute/hail).
 
-## Documentation
+Hail is pre-alpha software and under active development. If you're interested to get involved, see the [Development category](http://discuss.hail.is/c/development) of the discussion forum and chat with the Hail team and other developers on the [Hail development gitter](https://gitter.im/broadinstitute/hail-dev).
 
-Read the [docs](https://hail.is/docs/).
+We encourage use of the discussion forum for user and dev support, feature requests, and sharing your Hail-powered science. Please report any suspected bugs directly to [issues](https://github.com/broadinstitute/hail/issues).
```
Comment (cseed) [inline README.md]: Issues are now at hail-is/hail/issues.

**Pull request merged**